AA-416: Describe the RIP-7560 transaction receipt

forshtat opened the pull request.

forshtat committed:
AA-416: Describe the RIP-7560 transaction receipt

forshtat committed:
Add table of the transaction type with the fields' descriptions

drortirosh commented on RIPS/rip-7560.md:
```diff
@@ -94,6 +94,57 @@ Transactions of this type are referred to as
+| callGasLimit                  | QUANTITY       | Gas provided for the transaction execution frame                                    |
+| accessList                    | OBJECT         | An EIP-2930 compatible Access List structure                                        |
+| EIP-7702 authorizations (WIP) | ARRAY          | An EIP-7702 compatible list of contracts injected into EOAs                         |
+| authorizationData             | DATA           | Data that will be used by the Account to verify transaction                         |
```
update "signature" in this document to authorizationData

drortirosh commented on RIPS/rip-7560.md:
```diff
@@ -94,6 +94,57 @@ Transactions of this type are referred to as
+| paymaster                     | DATA, 20 Bytes | Address of the Paymaster contract (optional)                                        |
+| paymasterData                 | DATA           | Data that is provided to the Paymaster contract (if `paymaster` is set)             |
+| executionData                 | DATA           | Data that is provided to the Account contract for execution                         |
+| nonce                         | QUANTITY       | A 256 bit nonce. Use of `nonce > 2^64` is defined in RIP-7712                       |
```
separate optional "nonceKey" field

forshtat committed:
Merge branch 'master' of github.com:eth-infinitism/RIPs into AA-416-move-receipt-to-rip

forshtat committed:
Update: separate 'nonce' into 'nonceKey' and 'nonceSequence' in the RPC transaction object

Note that this was not done in the Solidity ABI encoding of a
transaction type, to maintain similarity to the ERC-4337.

drortirosh commented on RIPS/rip-7560.md:
```diff
@@ -106,7 +106,8 @@ The following table represents a full list of fields of an RIP-7560 transaction:
 | executionData                 | DATA           | Data that is provided to the Account contract for execution                         |
-| nonce                         | QUANTITY       | A 256 bit nonce. Use of `nonce > 2^64` is defined in RIP-7712                       |
+| nonceKey                      | QUANTITY       | A 192 bit nonce key. Use of `nonceKey != 0` is defined in RIP-7712.                 |
+| nonceSequence                 | QUANTITY       | A 64 bit nonce sequence. Use of `nonceKey != 0` is defined in RIP-7712.             |
```
I think its better to stick with "nonce" here. it will complicate implementation, as it removes a standard "nonce" field which is already in use.

forshtat committed:
Update

forshtat committed:
Split the 'nonce' into 'nonceKey' and 'nonceSequence' in the Solidity struct
